test/expected/hnsw_todo.out

```diff
@@ -44,8 +44,8 @@ SELECT ROUND(hamming_dist(v, '{0,0}')::numeric, 2) FROM small_world_ham ORDER BY
 -------
   0.00
   2.00
-  4.00
```

Comment:
Is there still anything wrong with this query? 
If not, it should be removed from `hnsw_todo.sql`  (and moved to hnsw_dist_func.sql unless an equivalent test is already there)
If something is still wrong, please add a relevant comment.